docs/README.md

```diff
@@ -0,0 +1,20 @@
+# Website
+
+This website is built using [Docusaurus](https://docusaurus.io/), a modern static website generator.
+
+
+## Local Development
+
+```bash
+pnpm start
+```
+
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+## Build
+
+```bash
+pnpm build
```

Comment:
Build intstructions are  not clear. Must be run in this folder!
npm install -g pnpm
pnpm install
pnpm build